src/lib/tests/MockRelayRenderer.tsx

```diff
@@ -4,10 +4,9 @@ import { QueryRenderer } from "react-relay"
 import {
   Environment,
   GraphQLTaggedNode,
-  OperationBase,
-  OperationDefaults,
+  INetwork as RelayNetwork,
```

Comment:
I wonder what their thinking was with this naming 1-off?

Reply:
I know, right? Bothers me. way. too. much.